magpie/utils.py

```diff
@@ -7,8 +7,7 @@
     HTTPOk, HTTPClientError, HTTPException, ConfigurationError, Configurator, Registry, Request, Response, truthy
 )
 from six.moves.urllib.parse import urlparse
-# noinspection PyProtectedMember
-from enum import EnumMeta
+from enum import EnumMeta  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "from enum import EnumMeta" should be placed before "from six.moves.urllib.parse import urlparse"](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)